File: dataset/python2java/python/CHECK_ARRAY_MAJORITY_ELEMENT.py
def f_gold ( a, n ) :
    mp = { }
    for i in range(n) :
        if a[i] in mp : mp [ a[i] ] += 1
        else : mp [ a[i] ] = 1
    for x in mp :
        if mp [ x ] >= len ( a ) // 2 :
            return True
    return False

File: result/translate/transcoder/python2java/java/CHECK_ARRAY_MAJORITY_ELEMENT.java
public static boolean f_gold ( int [ ] a , int n ) {
  int [ ] mp = new int [ n ] ;
  for ( int i = 0 ;
  i < n ;
  i ++ ) {
    if ( a [ i ] < mp [ i ] ) {
      mp [ a [ i ] ] ++ ;
    }
    else {
      mp [ a [ i ] ] = 1 ;
    }
  }
  for ( int x : mp ) {
    if ( mp [ x ] >= a . length / 2 ) {
      return true ;
    }
  }
  return false ;
}